sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md

```diff
@@ -0,0 +1,173 @@
+---
+title: SonarScanner for Azure DevOps
+url: /analysis/scan/sonarscanner-for-azure-devops/
+---
+
+
+[[info]]
+| By [SonarSource](https://www.sonarsource.com/) - GNU LGPL 3 - [Issue Tracker](https://jira.sonarsource.com/browse/VSTS) - [Source](https://github.com/SonarSource/sonar-scanner-vsts)  
+| **SonarScanner for Azure DevOps**
+
+The <!-- sonarqube -->[SonarQube](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarqube)<!-- /sonarqube --> <!-- sonarcloud -->[SonarCloud](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud)<!-- /sonarcloud --> extension for Azure DevOps <!-- sonarqube -->Server<!-- /sonarqube --> makes it easy to integrate analysis into your build pipeline. The extension allows the analysis of all languages supported by {instance}.
+
+<!-- sonarcloud -->
+Microsoft has published a [dedicated lab](https://aka.ms/sonarcloudlab) describing how to integrate Azure DevOps Pipelines and SonarCloud. The lab includes setting up a Branch Policy in Azure DevOps to block a Pull Request from being submitted if the changed code does not meet the quality bar.
+<!-- /sonarcloud -->
+
+## Compatibility
+Version <!-- sonarqube -->4.x<!-- /sonarqube --><!-- sonarcloud -->1.x<!-- /sonarcloud --> is compatible with:
+* TFS 2017 Update 2+
+* TFS 2018
+* Azure DevOps Server 2019
+* Azure DevOps
+
+<!-- sonarqube -->
+The SonarQube Extension embeds its own version of the [SonarScanner for MSBuild](/analysis/scan/sonarscanner-for-msbuild/).
+<!-- /sonarqube -->
+
+## Installation
+1. Install the extension [from the marketplace](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarqube). 
```

Comment:
Needs a separate link for SonarCloud:
https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud